client/src/modules/report-group/index.html

```diff
@@ -0,0 +1,170 @@
+<div class="flex-header static">
```

Comment:
As I understand it these files should **not** be included in this pull request - they belong to https://github.com/IMA-WorldHealth/bhima-2.X/pull/1778.